data/apis/api3/apiMetadata.json

```diff
@@ -2,5 +2,6 @@
   "name": "API3",
   "active": true,
   "airnode": "0xA3121b14485DAa8c797178Cb21445b0f30e75178",
-  "xpub": "xpub661MyMwAqRbcFeZ1CUvUpMs5bBSVLPHiuTqj7dZPertAGtd3xyTW1vrPspz7B34A7sdPahw7psrJjCXmn8KpF92jQssoqmsTk8fZ9PZN8xK"
+  "xpub": "xpub661MyMwAqRbcFeZ1CUvUpMs5bBSVLPHiuTqj7dZPertAGtd3xyTW1vrPspz7B34A7sdPahw7psrJjCXmn8KpF92jQssoqmsTk8fZ9PZN8xK",
+  "logoPath": "https://amberdata.io/logos/amberdata_logo_bug_sticker.svg"
```

Comment:
Is this logo correct? It might be I just can't see anything that says Amberdata either in this file or in the `api3` directory.

Reply:
In production `api3` won't exist as it's the equivalent of a third-party oracle which is against the business model of API3. The logo is just for demonstration purposes. I should have noted that. I'll update it with something more appropriate shortly.

Reply:
I've updated the logos :)

Reply:
We will need to have the airseeker `xpub` in this repo somewhere though